records_mover/db/driver.py

```diff
@@ -21,42 +22,33 @@
 logger = logging.getLogger(__name__)
 
 
-class DBDriver:
-    def __init__(self,
-                 db: Union[sqlalchemy.engine.Engine,
-                           sqlalchemy.engine.Connection], **kwargs) -> None:
-        self.db = db
-        self.db_engine = db.engine
-        self.meta = MetaData()
-
-    def best_scheme_to_load_from(self) -> str:
-        return 'file'
-
-    def has_table(self, schema: str, table: str) -> bool:
-        return self.db.dialect.has_table(self.db, table_name=table, schema=schema)
-
-    def table(self,
-              schema: str,
-              table: str) -> Table:
-        return Table(table, self.meta, schema=schema, autoload=True, autoload_with=self.db_engine)
-
-    def schema_sql(self,
-                   schema: str,
-                   table: str) -> str:
-        """Generate DDL which will recreate the specified table and return it
-        as a string.  Returns None if database or permissions don't
-        support operation.
+class NegotiatesLoadFormat(metaclass=ABCMeta):
+    def can_load_this_format(self, source_records_format: BaseRecordsFormat) -> bool:
+        """Return true if the specified format is compatible with the load()
+        method"""
+        return False
 
+    # TODO: Is this still used?  Is this for load or unload or both?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker